app/interactors/interface/customer_base.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'concerns/logging'
+
+module Interface
+  class CustomerBase
+    include Interactor
+    include Interface::Logging
+
+    delegate :customer, to: :entity
```

Comment:
We don't need the delegates if we're passing the customer to `log!`

Suggest changing this naming from `customer` to `interfaceable`.